packages/@cdktf/cli-core/src/lib/models/terraform-cli.ts

```diff
@@ -49,6 +61,27 @@ export class TerraformCli implements Terraform {
       args.push("-upgrade");
     }
 
+    // TODO: do we need to run this if we use the TF CLI to run in TFC?
```

Comment:
Yes, if we don't do this, Remote execution starts failing with:

<img width="667" alt="image" src="https://user-images.githubusercontent.com/573531/206459322-ffac48cc-9463-497f-88f1-1b99d30f9cd7.png">

Reply:
Damn.. We currently have no way to know whether something runs in TFC with remote execution 😬

Reply:
I haven't seen a problem with local exec using this.

Reply:
Nope, but it downloads three provider binaries, so I do notice this on my slow German internet connection 😅

Reply:
Or at least something network heavy seems to be happening – I'd love to be wrong here 😄

Reply:
fixed 👍 (this was also caught by multiple integration tests, after we got them running)